### DISCUSSION TITLE
Commits from other repositories contain wrong references on GitHub

### DISCUSSION THREAD

**Discussion by darius-m** [keycloak/keycloak#33124] [Feedback]
Some commits (e.g. https://github.com/keycloak/keycloak/commit/ae6a686870b57182c0635ae60f0fe81d57cf15e3, https://github.com/keycloak/keycloak/commit/9f37a83c517f5f5e1664dd9a3b74a4be5ef16b11) contain issue / pull request references from other repositories, which when imported into the Keycloak repository link to the wrong discussion. In particular, for security advisories / CVEs the references would more appropriately link to the relevant issue description (advisory / CVE).

CC @stianst

**Comment by jonkoops** [comment]
These commits are cherry-picked from a private fork where we handle CVEs. I think we could start putting in some CVE numbers in the commits themselves perhaps.

**Reply by stianst** [reply]
Yes, as we have agreed they should not have links to GitHub Issues, but instead use the CVE number for easy identifying the commits later.
